.coveragerc

```diff
@@ -0,0 +1,5 @@
+[run]
+omit =
+    src/Zoomba/Helpers/options.py
+    src/Zoomba/Helpers/chromeOptions.py
+    src/Zoomba/Helpers/EdgePlugin.py
```

Comment:
Omitting these as they are really just there as a temporary bridge while we wait for selenium 4 and support from AppiumLibrary for selenium 4